typing/untypeast.ml

```diff
@@ -348,6 +348,7 @@ let pattern : type k . _ -> k T.general_pattern -> _ = fun sub pat ->
           match tyo, arg with
           | Some ty, Some arg ->
               Some (vl, Pat.mk ~loc (Ppat_constraint (arg, ty)))
+          | None, Some arg -> Some (vl, arg)
```

Comment:
Another option would be to use `Some ([], arg)` in this case. Or maybe the list `vl` of existential quantifiers could be moved under the option in the code above.

Reply:
I moved it under the option.

Reply:
And made an mistake :-)